lib/confirm-controller.js

```diff
@@ -1,61 +1,129 @@
 'use strict';
 
-const BaseController = require('./base-controller');
 const _ = require('lodash');
+const i18nLookup = require('i18n-lookup');
+const Mustache = require('mustache');
```

Comment:
apparently Mustache isn't a dependency of Hogan - make me out a liar! We can't pass the raw function and use i18n-lookup with the shorthand if we use hogan though. I'll see if Mustache appears in the dependency tree without this change